tests/datatypes/test_frozenset.py

```diff
@@ -2,7 +2,33 @@
 
 
 class FrozensetTests(TranspileTestCase):
-    pass
+    def test_creation(self):
+        # Empty dict
+        self.assertCodeExecution("""
+            x = frozenset()
+            print(x)
+            """)
+
+        # Set constant
+        self.assertCodeExecution("""
+            x = {'a'}
+            print(x)
+            """)
+
+        self.assertCodeExecution("""
+            x = frozenset(['a'])
+            print(x)
+            """)
+
+        self.assertCodeExecution("""
+            x = frozenset((1, 2))
+            print(x)
+            """)
+
+        self.assertCodeExecution("""
+            x = frozenset('a')
+            print(x)
+            """)
```

Comment:
Please combine all these tests into one large `self.assertCodeExecution()`, it's ok to keep assigning the value of x and then accessing it.